### ISSUE TITLE
Shadow fails to draw when from/to min/max has an integer value of 0

### ISSUE THREAD

**Issue by hinshun** [IonDen/ion.rangeSlider#193]
My use case:
* Two sliders that affect each other's `from_min` and `from_max`.
* Data-bindings to values that calls update on ion.rangeSlider to redraw shadows

The problem is that when `from_shadow` is enabled, and that a slider has a range from 0 to x, updating `from_min` to an integer value of 0 hides the shadow. This is jarring from a UX perspective because when you have a moving shadow and the shadow suddenly disappears, the slider seems like its range of motion is completely locked.

This happens because of the following code from `drawShadow()`:
```javascript
if (o.type === "single") {
    if (o.from_shadow && (o.from_min || o.from_max)) {
        from_min = this.calcPercent(o.from_min || o.min);
        from_max = this.calcPercent(o.from_max || o.max) - from_min;
        // ...
```
If `o.from_min` has a integer value of 0, it will evaluate as false despite the intention of the developer for there to be a value (otherwise it should be null).

**Comment by IonDen** [comment]
Hello, i will check this soon. Next week i think.

**Comment by IonDen** [comment]
Hi, should be fixed in 2.0.5. Please check
